src/test/java/com/dynatrace/openkit/protocol/BeaconTest.java

```diff
@@ -1118,7 +1118,7 @@ public void sessionIDIsAlwaysValue1OnDataCollectionLevel0() {
         target.setBeaconConfiguration(new BeaconConfiguration(1, DataCollectionLevel.OFF, CrashReportingLevel.OFF));
 
         //when
-        int sessionID = target.getSessionNumber(SESSION_ID);
+        int sessionID = target.getSessionNumber();
```

Comment:
rename for consistency reasons (below `sessionID` was renamed to `sessionNumber`)